qmri_megre/MEGRE.json

```diff
@@ -0,0 +1,8 @@
+{
+
+	"MagneticFieldStrength": "3",
+	"Manufacturer": "Siemens",
+	"ManufacturerModelName": "TrioTim",
+	"PulseSequenceType": "GR",
+	"EchoTime": "0.02"
```

Comment:
I think the `EchoTime` should not be in this top level file for `MEGRE` as it changes.